tgui/packages/tgui/interfaces/PersonalCrafting.tsx

```diff
@@ -714,6 +714,28 @@ const RecipeContentCompact = ({ item, craftable, busy, mode }) => {
                       })
                     }
                   />
+                  {item.mass_craftable && (
+                    <Button
+                      my={0.3}
+                      lineHeight={2.5}
+                      align="center"
+                      content="Make a lot"
```

Comment:
As a UI formatting thing:

Rather the two buttons on top of each other, I think it might be a lot nicer formatted if, we formatted it something like: 

![image](https://github.com/tgstation/tgstation/assets/51863163/ee67c4c6-5b39-4357-adff-3301e7579424)

Where above is non-loopable things and below is loopable things. 

And where the loop button (just represented by an icon) has a tooltip explaining what it's doing. IE, "Repeat this craft until you run out of ingredients."